cmd/cli/main.go

```diff
@@ -24,12 +21,21 @@ type DAST struct {
 	AuthHeaders      []string `arg:"--header" help:"List of headers for the DAST agent to authenticate with your API"`
 }
 
-type args struct {
-	DAST *DAST `arg:"subcommand:dast" help:"Test the given app for bugs and vulnerabilities"`
+type LocalScan struct {
```

Comment:
can we just include a boolean field in the DAST struct which is set by a --local flag